trio/_core/tests/tutil.py

```diff
@@ -97,7 +97,11 @@ def restore_unraisablehook():
 
     @contextmanager
     def disable_threading_excepthook():
```

Comment:
@graingert, I think in 3.10+ this could be called `restore_threading_excepthook()`, yes?  I'm lazy about refactoring it now but maybe it'd be correct?  Correct in the future looking sense...

Reply:
Yep

Reply:
Meh, no need to expand this PR.  If I do rename I'll make another PR for it.